.github/workflows/test-native-gradle-plugin.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: "🔧 Install GraalVM (dev)"
         uses: graalvm/setup-graalvm@main
         with:
-          java-version: 'dev'
+          java-version: '22.0.1'
```

Comment:
Also, why this change?

Reply:
- This is related to the issue I pointed out at https://github.com/graalvm/native-build-tools/pull/603#pullrequestreview-2118567359 . Under GraalVM CE For JDK24 Dev, this CI fails as https://github.com/graalvm/native-build-tools/actions/runs/9517556127/job/26236415815?pr=603 . This CI works fine under GraalVM CE For JDK22, as shown in https://github.com/graalvm/native-build-tools/actions/runs/9520173469/job/26244879976?pr=603 . I don't know anything about the changes after JDK23. 
- I have now rolled back the changes to show the CI.